packages/server/beaver_triple_service/go.mod

```diff
@@ -14,6 +14,12 @@ require (
 	gopkg.in/yaml.v3 v3.0.1
 )
 
+require (
+	github.com/inconshreveable/mousetrap v1.1.0 // indirect
+	github.com/spf13/cobra v1.6.1 // indirect
+	github.com/spf13/pflag v1.0.5 // indirect
+)
+
 require (
 	github.com/golang/protobuf v1.5.0 // indirect
```

Comment:
Together with the require clause below.

Reply:
Corrected with [e618c5c](https://github.com/acompany-develop/QuickMPC/pull/123/commits/e618c5ca260670501207788c42d95a52ba65bb6a)